randovania/games/dread/json_data/Cataris.txt

```diff
@@ -1185,7 +1193,9 @@ Extra - asset_id: collision_camera_015
   > Door to Artaria Transport Access
       All of the following:
           Morph Ball
-          Gravity Suit or Lava Damage ≥ 600
+          Any of the following:
+              Gravity Suit
+              Heat/Cold Runs (Intermediate) and Lava Damage ≥ 1000
```

Comment:
Changed to 1000 here?

Reply:
Yes, because the opposite connection is 1000. This direction would also be slower with having to unmorph and regrab into every tunnel